docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -2,19 +2,19 @@
 
 = Verifying the {image_service} adoption
 
-Verify that you successfully adopted your {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
+Verify that you adopted the {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
 
 .Procedure
 
-. Test the {image_service_first_ref} from the {rhos_prev_long} CLI. You can compare and make sure the configuration has been correctly applied to the {image_service} pods:
+. Test the {image_service} from the {rhos_prev_long} command line interface (CLI). You can compare and ensure that the configuration is applied to the {image_service} pods:
```

Comment:
Step 1: Can you clarify what are we comparing?

Reply:
I think this refers to the resulting configuration that is applied to the Pods, but tbh I would  compare it with the applied Glance CR where we explicitly inject the configuration!
@jistr do you have any hint on this part?

Reply:
Jiri is on PTO until Aug 25th. Is there someone else who would know? Or we could table it for now and clarify in another PR when he comes back.

Reply:
I agree to check this on a different PR. 
I read about the tool and this seems something that the human operator should do: in general the human operator needs to compare the 17.x Glance configuration with the one provided in the resulting/adopted Pod, and make sure it's aligned to what we had previously.
I don't see this one as a critical step for Glance, because most of the defaults are provided by the `glance-operator` and there might be different defaults compared to the previous release (e.g. glance-direct), so I prefer to focus on this detail on a different PR.

Reply:
I can see you added _command line interface_, it might not be needed here:  the IBM style guide says "Do not spell out an abbreviation that is commonly known to the target audience."